engine-tests/src/utils/solidity/exit_precompile.rs

```diff
@@ -94,7 +94,9 @@ impl Tester {
         match result.status {
             aurora_engine::parameters::TransactionStatus::Succeed(_) => Ok(result),
             aurora_engine::parameters::TransactionStatus::Revert(_bytes) => Err(Revert),
-            other => panic!("Unexpected status {other:?}"),
+            aurora_engine::parameters::TransactionStatus::Error(err) => {
+                panic!("Unexpected status {err:?}")
```

Comment:
```suggestion
                panic!("Unexpected status {:?}", err.as_ref());
```
?

Reply:
I decided that may be it will be more informative. As it'll return something like: `Error(GasOverflow)` but I can change it, obviously, if you think it will be more clear.